src/utils/parse-raw-flags.js

```diff
@@ -12,12 +12,12 @@
 
 const parseRawFlags = function (raw) {
   const rawFlags = raw.reduce((acc, curr, index, array) => {
-    if (curr.input.match(/^-{1,2}/)) {
+    if (/^-{1,2}/.test(curr.input)) {
```

Comment:
[sand] Tiny performance improvement (also applies below):

```js
    if (curr.input.startsWith('-')) {
```

Reply:
I'm going to re-visit the usage of this function as I think we can remove it all together.